guides/writing-addons/index.md

```diff
@@ -33,6 +33,17 @@ The default way to make a component is to put the implementation in `addon/`, wh
 
 Fortunately, when we run `ember generate component my-component-name` in an addon project, the CLI takes care of all this re-exporting business. It creates the necessary files and code for us. Addon authors don't usually need to think about the `app` directory or do any work in it.
 
+#### `public/`
+
+You can add static assets in the `public/` directory in your addon, and they will automatically be
+merged into the public directory in the host app under `/your-addon-name/`.
+For example, if you add an image at `public/icon.png`, the host app can reference it as
```

Comment:
Can you be a bit more concrete in the examples? Something like "For example, if you have an addon named X and an image at public/icon, then you can reference it as Y in the application"